src/cmd/src/bin/compute_node.rs

```diff
@@ -14,12 +14,10 @@
 
 #![cfg_attr(coverage, feature(no_coverage))]
 
-use global_stats_alloc::INSTRUMENTED_JEMALLOC;
```

Comment:
Is this crate still used?

Reply:
No, I will remove it later along with other deprecated crate